packages/components/badge/src/BadgeItem.styles.tsx

```diff
@@ -0,0 +1,43 @@
+import { makeVariants } from '@spark-ui/internal-utils'
+import { cva, type VariantProps } from 'class-variance-authority'
+
+import { DEFAULT_INTENT, DEFAULT_SIZE, DEFAULT_TYPE } from './config'
+
+export const styles = cva(
+  ['inline-flex h-fit', 'empty:p-none', 'text-center font-bold', 'border-md rounded-full'],
+  {
+    variants: {
+      intent: makeVariants<'intent', ['error', 'info']>({
+        error: ['bg-error', 'text-on-error'],
+        info: ['bg-info', 'text-on-info'],
+      }),
+      size: makeVariants<'size', ['sm', 'md']>({
+        sm: ['text-small', 'px-[var(--sz-6)] py-[var(--sz-2)]', 'empty:h-sz-12 empty:w-sz-12'],
```

Comment:
I couldn't make it work with the standard padding sizes 😢

Reply:
Then I think either:
- the mockups should be updated
- OR we were supposed to use width/height instead of padding

Reply:
It should be the first option, because working with width is not valid for this component since it can grow depending on its inner content (1, 99, 99+...). Anyway, I'll check it with design/UX after all the technical code review...

Reply:
I think that we should add sizes for `padding`, `margin`, etc. When I tried spark for building a website it was a handicap